content/docs/how-we-work/release-versioning.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Release Versioning
+title: Release Versioning and Planning
```

Comment:
@Yemaneberhan-Lemma I think the title needs revisiting as I added some content about planning the release